markdown/clientaccess/client_auth.html.md.erb

```diff
@@ -99,27 +99,25 @@ This example shows how to edit the `pg_hba.conf` file of the master to allow rem
 
 HAWQ allocates some resources on a per-connection basis, so setting the maximum number of connections allowed is recommended.
 
-To limit the number of active concurrent sessions to your HAWQ system, you can configure the `max_connections` server configuration parameter on master or the `seg_max_connections` server configuration parameter on segments. These parameters are *local* parameters, meaning that you must set them in the `hawq-site.xml` file of all HAWQ instances.
+To limit the number of active concurrent sessions to your HAWQ system, you can configure the `max_connections` server configuration parameter. This parameter is a *local* parameter, meaning that you must set them in the `hawq-site.xml` file of all HAWQ instances.
 
-When you set `max_connections`, you must also set the dependent parameter `max_prepared_transactions`. This value must be at least as large as the value of `max_connections`, and all HAWQ instances should be set to the same value.
+When you set `max_connections`, you can also set the value for the maximum concurrent connections for the segments by using `seg_max_connections`. Setting this parameter overrides the value of `max_connections` on the segment instances.
 
 Example `$GPHOME/etc/hawq-site.xml` configuration:
 
 ``` xml
   <property>
       <name>max_connections</name>
-      <value>500</value>
+      <value>200</value>
```

Comment:
It's ok, but I think better to be 1280 as the default value we used.

Reply:
I find we mentioned the best practice setting is 1280, so I agree with Radar if we can directly set it to 1280 to reflect what we recommended.

Reply:
I am ok with default value changes, as Radar said, if we mentioned the best practice value as 1280 for this guc, maybe it is better to directly change to 1280 to reflect what we recommended. Thanks.